sonic-xcvrd/setup.py

```diff
@@ -10,9 +10,10 @@
     url = 'https://github.com/Azure/sonic-platform-daemons',
     maintainer = 'Kebo Liu',
     maintainer_email = 'kebol@mellanox.com',
-    entry_points = {
+    packages = find_packages()
+    entry_points={
```

Comment:
It wasn't the spaces, it is a missing comma at the end of the previous line.

Reply:
Thanks for pointing, fixed